code/distance-between-vectors/src/main/java/ru/unn/agile/distanceBetweenVectors/model/VectorMetric.java

```diff
@@ -0,0 +1,34 @@
+package ru.unn.agile.distanceBetweenVectors.model;
+
+import java.util.Collections;
+import java.util.Vector;
+
+public class VectorMetric {
+    private float sumOfModules = 0f;
```

Comment:
What if we run calculateL1 and next calculateL2? Please, test it, I think, answer will be incorrect